README.md

```diff
@@ -1,5 +1,9 @@
 # yosh - Your Own SHell
 
+This is fork from https://github.com/supasate/yosh
+
+And it's modifed by Ted,fixed some problems and made it more looks like a noraml shell
+
```

Comment:
Don't need to specify this comment. I'll add detail and contributor into CHANGES.md file.